### DISCUSSION TITLE
[Request] LXC or VM script to install Nextcloud

### DISCUSSION THREAD

**Discussion by rcv11x** [tteck/Proxmox#1633] [Ideas]
Currently we have the NextCloudPi LXC script but it would be a good idea to create a LXC or VM script to install a Nextcloud server, would it be possible to add it? 

PS: I think it would have better performance in LXC correct me if I'm wrong, thanks

**Comment by OsipovOV** [comment]
https://www.youtube.com/watch?v=vOehK7n4MMs
